diff --git a/node_modules/photoswipe/dist/photoswipe-lightbox.esm.js b/node_modules/photoswipe/dist/photoswipe-lightbox.esm.js
index e5d4d8b..6181ec5 100644
--- a/node_modules/photoswipe/dist/photoswipe-lightbox.esm.js
+++ b/node_modules/photoswipe/dist/photoswipe-lightbox.esm.js
@@ -637,7 +637,7 @@ class Content {
         const placeholderSrc = this.instance.applyFilters('placeholderSrc', // use  image-based placeholder only for the first slide,
         // as rendering (even small stretched thumbnail) is an expensive operation
         this.data.msrc && this.slide.isFirstSlide ? this.data.msrc : false, this);
-        this.placeholder = new Placeholder(placeholderSrc, this.slide.container);
+        this.placeholder = new Placeholder(this.data.src, this.slide.container);
       } else {
         const placeholderEl = this.placeholder.element; // Add placeholder to DOM if it was already created
 
